release-notes.txt

```diff
@@ -1,3 +1,503 @@
+RIOT-2025.01 - Release Notes
+============================
+RIOT is a multi-threading operating system which enables soft real-time
+capabilities and comes with support for a range of devices that are typically
+found in the Internet of Things: 8-bit and 16-bit microcontrollers as well as
+light-weight 32-bit processors.
+
+RIOT is based on the following design principles: energy-efficiency, soft
+real-time capabilities, small memory footprint, modularity, and uniform API
+access, independent of the underlying hardware (with partial POSIX compliance).
+
+RIOT is developed by an international open-source community which is
+independent of specific vendors (e.g. similarly to the Linux community) and is
+licensed with a non-viral copyleft license (LGPLv2.1), which allows indirect
+business models around the free open-source software platform provided by RIOT.
+
+
+About this release
+==================
+
+The 2025.01 release includes:
+
+## List of notable changes here ##
+
+144 pull requests, composed of 280 commits, have been merged since the
+last release, and 0 issues have been solved. 32 people contributed with
+code in 65 days. 665 files have been touched with 75661 (+) insertions and
+6329 deletions (-).
+
+
+Notations used below
+====================
+
+    + means new feature/item
+    * means modified feature/item
+    - means removed feature/item
+
+
+New features and changes
+========================
```

Comment:
`GOVERNANCE.md` is not mentioned, as far as I can see.

Reply:
That would also be the spotlight feature to point out first in the release notes IMO.

Reply:
Done